FrontPage/AppDelegate.swift

```diff
@@ -38,6 +40,32 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
   
   func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
     apollo.cacheKeyForObject = { $0["id"] }
+    
+    // bootstrap the registration process by asking the user to 'Accept' and then register with APNS thereafter
+    let settings = UIUserNotificationSettings(types: [.alert, .badge, .sound], categories: nil)
+    UIApplication.shared.registerUserNotificationSettings(settings)
+    UIApplication.shared.registerForRemoteNotifications()
```

Comment:
Move the above code into its own function and give it an appropriate name as it will make it easier for others to read the code.

Reply:
Ok, should the new function be moved out of the AppDelegate file?